kolibri/core/assets/src/utils/browser.js

```diff
@@ -39,9 +39,9 @@ export function isEmbeddedWebView() {
 }
 
 export function isAppView() {
```

Comment:
I think renaming this to `isAppContext` as well, for consistency, would be good.

Reply:
Done